.github/autobuild/mac.sh

```diff
@@ -60,7 +62,7 @@ build_app_as_dmg_installer() {
     if prepare_signing; then
         BUILD_ARGS=("-s" "${MACOS_CERTIFICATE_ID}")
     fi
-    ./mac/deploy_mac.sh "${BUILD_ARGS[@]}"
+    TARGET_ARCH="${TARGET_ARCH}" ./mac/deploy_mac.sh "${BUILD_ARGS[@]}"
```

Comment:
Didn't read the code further, but would it be possible to pass the arch by normal arguments, not environment variables to stay consistent?